setup_preprocessing.py

```diff
@@ -34,6 +34,8 @@
     install_requires=required_deps,
     extras_require=extras,
     python_requires=">=3.5",
-    packages=find_packages(include=["*elasticdl_preprocessing*"],),
+    packages=find_packages(
+        include=["elasticdl_preprocessing*"], exclude=["*test*"]
+    ),
     package_data={"": ["requirements.txt"]},
```

Comment:
elasticdl_preprocessing/requirement.txt

Reply:
After verification, if update to `requirements.txt`, this file won't be included in the wheel.

Reply:
IMO, we should package the `requirements.txt` into the wheel.

Reply:
> IMO, we should package the `requirements.txt` into the wheel.

Yes, in the current implementation, it's packaged.